### PR TITLE
Update wacom-tablet from 6.3.45-2 to 6.3.45-3

### PR DESCRIPTION
**Pull request by i0ntempest**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by i0ntempest** [comment]
There is a launchctl job with a random UUID in its label and is impossible to unload using homebrew.

**Comment by i0ntempest** [comment]
Already tried using a wildcard, won’t work.

**Comment by bevanjkay** [comment]
@i0ntempest Is the UUID randomised on every install?

**Comment by i0ntempest** [comment]
> @i0ntempest Is the UUID randomised on every install?

Seems like it. I tried a few times and the check reported a different UUID every time.

**Comment by neutric** [comment]
> > @i0ntempest Is the UUID randomised on every install?
> 
> Seems like it. I tried a few times and the check reported a different UUID every time.

Isn't the CI VM/container created from scratch on every run? If so, it could be the Hardware UUID instead of being random.

File: Casks/wacom-tablet.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@
               "com.wacom.DisplayMgr",
               "com.wacom.RemoveWacomTablet",
               "com.wacom.TabletDriver",
+              "com.wacom.UpdateHelper",
```
Comment (neutric) [inline Casks/wacom-tablet.rb:24]: `com.wacom.UpgradeHelper.53252.934882E6-00C7-4B04-8F5F-D3E2A98273E0`
^Up**grade**Helper, not UpdateHelper

Comment (i0ntempest) [reply Casks/wacom-tablet.rb:24]: UpgradeHelper and UpdateHelper are two different things, they both exist in this package

Comment (miccal) [reply Casks/wacom-tablet.rb:24]: Just bear with me while I try a few things, please.

Comment (miccal) [reply Casks/wacom-tablet.rb:24]: Fixed.

**Pull request merged**